README.md

```diff
@@ -210,6 +210,12 @@ Haskell's [groupBy](http://zvon.org/other/haskell/Outputlist/groupBy_f.html).
 
 **Note:** Intersperse can be used with JSX components! Specially with the `differentLast` iterator:
 
+### replaceElementBy
+`(fn(array_element), value, array) -> array` Replaces an element in an array with `value` based on the boolean result of a function `fn`.
+
+### replaceElement
```

Comment:
these docs were added in the middle of `intersperse`'s docs

Reply:
Fixed